Dockerfile

```
@@ -66,35 +91,19 @@ RUN apt-get remove --quiet --quiet $INSTALL_DEPS
 RUN apt-get --quiet --quiet clean
 RUN rm -rf /var/lib/apt/lists/*
 
-
-###
-# Setup the user and its home directory
-###
-FROM install AS setup_user
-
 ###
-# Create unprivileged user
+# Setup working directory and entrypoint
 ###
-RUN groupadd -r $USER
-RUN useradd -r -c "$USER user" -g $USER $USER
 
 # Put this just before we change users because the copy (and every
 # step after it) will always be rerun by docker, but we need to be
 # root for the chown command.
-COPY . $HOME
-RUN chown -R ${USER}:${USER} $HOME
-
-
-###
-# Setup working directory and entrypoint
-###
-FROM setup_user AS final
+COPY src ${CISA_HOME}
```

I don't think we need `version.txt` or the contents of the repository's `secrets/` directory.

```suggestion
COPY src/scan.sh ${CISA_HOME}
```